structs/series.go

```diff
@@ -32,4 +34,59 @@ type SeriesStruct struct {
 	Performance     SeriesPerformanceStruct `json:"performance,omitempty"`
 	SportsbookOdds  []SportsbookOddsStruct  `json:"sportsbook_odds"`
 	Chain           *[]int64                `json:"chain"`
+	Summary         SeriesSummary           `json:"summary"`
+}
+
+// avoid recursion when unmarshaling
+type seriesStruct SeriesStruct
+
+// We need to unmarshal summary into the game-specific struct
+func (s *SeriesStruct) UnmarshalJSON(data []byte) error {
+	// find the outer-most keys
+	var partial map[string]json.RawMessage
+	if err := json.Unmarshal(data, &partial); err != nil {
+		return err
+	}
+	summary := partial["summary"]
+
+	// This is not strictly necessary but it is faster
+	delete(partial, "summary")
+	data, _ = json.Marshal(partial)
+
+	var ss seriesStruct
+	if err := json.Unmarshal(data, &ss); err != nil {
+		return err
+	}
+
+	// "null" is 4 bytes
+	if len(summary) > 4 {
```

Comment:
Maybe explicitly check for "null" instead?

Reply:
There can be no useful data that has `len <= 4` anyways so my thought process was that it doesn't matter if it's explicitly set to "null". This way there is no need for a string comparison (although the performance difference is most likely negligible).